src/openrct2/windows/staff.c

```diff
@@ -290,7 +290,7 @@ uint32 window_staff_page_enabled_widgets[] = {
 	(1 << WIDX_TAB_3)
 };
 
-static const uint32 staffCostumeNames[] = {
+const rct_string_id StaffCostumeNames[] = {
```

Comment:
I'm not sure this should be defined in a window definition. The peep/staff.c file is a better option IMO.